CMakeLists.txt

```diff
@@ -94,23 +47,29 @@ add_library(${POCLIB} INTERFACE)
 # Make an alias (useful for in-tree builds)
 add_library(${POCLIB}::${POCLIB} ALIAS ${POCLIB})
 
-target_compile_definitions(${POCLIB}
-INTERFACE
-  ${PORTABILITY_STRATEGY}
-)
+# make cache variables for install destinations
+include(GNUInstallDirs)
 
 # Enables
 # #include <ports-of-call/portability.hpp>
 target_include_directories(${POCLIB}
   INTERFACE
-  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
-  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
 )
 
+target_compile_features(${POCLIB}
+  INTERFACE
+    cxx_std_14
```

Comment:
Doesn't have to be, my thinking is this would be more consistent with `singularity-eos`, but it also makes sense that it's not a requirement that needs to be propagated.
Although, as we discussed the other day, if we want to have some variadic implementation of `PortablMDarray`, having 14 features would be useful

Reply:
So should I interpret that as a minimum cxx standard? My main question is will this break builds where I try to enforce c++ 17?

Reply:
> So should I interpret that as a minimum cxx standard? My main question is will this break builds where I try to enforce c++ 17?
No, see https://cmake.org/cmake/help/latest/manual/cmake-compile-features.7.html#requiring-language-standards. Note that it would be possible to select individual features here (e.g. `cxx_variadic_templates`), although the utility of that isn't clear if we need to support a broad range of compilers (i.e., compilers can be expected to provide all features of a selected standard, but some compilers my provide individual advanced features to previous standards. GNU likes to do this).